bazelrio/libraries/cpp/wpilib/hal/BUILD

```diff
@@ -17,3 +17,16 @@ cc_library(
         ],
     }),
 )
+
+cc_library(
```

Comment:
Nit: You could make the argument that this should be only on the java side, since it only applies to java.

I wouldn't change it now if you have it working. Shouldn't be user facing so we could refactor if we wanted to later